Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -1644,7 +1644,7 @@ private void OnRTCRoomUpdateReceivingCompleted(ref UpdateReceivingCallbackInfo d
 
             foreach(LobbyMember lobbyMember in CurrentLobby.Members)
             { 
-                if(lobbyMember.ProductId == data.LocalUserId)
+                if(lobbyMember.ProductId != data.ParticipantId)
```

Comment:
For this change in particular, someone with more experience in the lobby area of the project will need to weigh in - I don't feel like I can approve or request changes on this one. But we will continue to look into it, and keep yah posted :)

Reply:
Ok.

https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/a31af6e2b7b2b736d3db8e5fe37dbdce38487ffd/Assets/Plugins/Source/EOS_SDK/Generated/RTCAudio/UpdateReceivingCallbackInfo.cs#L24-L27

https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/a31af6e2b7b2b736d3db8e5fe37dbdce38487ffd/Assets/Plugins/Source/EOS_SDK/Generated/RTCAudio/UpdateReceivingCallbackInfo.cs#L34-L37

LocalUserId is the ID of oneself, and ParticipantId is the ID of the user who is the target of the change.